sample/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,14 @@
+<resources>
+  <string name="app_name">Mapzen Turn By Turn Sample</string>
```

Comment:
Even though the sample app is configured to use Mapzen Turn-by-Turn as the backend by default I'm thinking we want to call it "On the Road Sample" or just "On the Road" since in theory it can be used with any instance of Valhalla. Thoughts @sarahlensing @mjcunningham?